DeUrgenta/Features/Menu/View/MenuView.swift

```diff
@@ -0,0 +1,70 @@
+import SwiftUI
+
+protocol MenuViewDelegate {
+    func menuViewShouldRedirect(from view: MenuView, withItem item: MenuOption)
+    func menuViewDidTapDonateButton(_ view: MenuView)
+}
+
+struct MenuView: View {
+    @ObservedObject var viewModel: MenuViewModel
+    var delegate: MenuViewDelegate
+
+    var body: some View {
+        VStack(spacing: 0) {
+            ScrollView {
+                HStack {
+                    Image("logo")
+                        .resizable()
+                        .frame(width: 100, height: 100)
+
+                    Spacer()
+                }
+                .padding()
+
+                ForEach(viewModel.options, id: \.id) { option in
+                    VStack {
+                        Button(action: {
+                            delegate.menuViewShouldRedirect(from: self, withItem: option)
+                        }, label: {
+                            VStack(alignment: .leading) {
+                                Text(option.name)
+                                    .font(.custom("IBMPlexSans-Bold", size: 16))
```

Comment:
Maybe we can better define custom fonts to avoid miss typing.

Reply:
Yes, of course